cpp/lib/src/ZoteroHarvesterConversion.cc

```diff
@@ -993,7 +995,7 @@ void GenerateMarcRecordFromMetadataRecord(const MetadataRecord &metadata_record,
             subfields.appendSubfield('c', creator.title_);
```

Comment:
Indentation is not consistent with the following line.

Reply:
Looks correct in editor